gdcdictionary/schemas/program.yaml

```diff
@@ -36,6 +36,7 @@ properties:
   name:
     type: string
     description: "Full name/title of the program."
+    pattern: "^[A-Z0-9]+|[0-9]+|[A-Z]+$"
```

Comment:
Just to clarify, I need to add the $ symbol to the other 2 options, to signify that no other trailing characters are allowed.